test/over_react_redux/store_bindings_tests.dart

```diff
@@ -358,18 +358,19 @@ void sharedSelectorHookAndConnectTests<T>(
   });
 }
 
-class TestState<T> {
+class TestState<T extends Object?> {
   final T interestingValue;
   final int otherValue;
 
   TestState({
-    @required this.interestingValue,
-    @required this.otherValue,
+    required this.interestingValue,
+    required this.otherValue,
   });
 
   TestState<T> update({
-    T interestingValue,
-    int otherValue,
+    // FIXME this doesn't allow updating to null
+    T? interestingValue,
+    int? otherValue,
```

Comment:
Does this have a follow-up ticket # that should go here?

Reply:
Good catch; addressed in https://github.com/Workiva/over_react/pull/856/commits/b123072b8b841d1b21d11755efbd83f41c25aed8